.github/mergify.yml

```diff
@@ -13,13 +13,13 @@ pull_request_rules:
     - label!=needs-rebase
     - check-success=DCO
 
-    # If files are changed in .github/, the actionlint check must pass
+    # If workflow configuration files in .github/ are changed, the actionlint check must pass
     - or:
       - and:
         # regex should match the one in .github/workflows/actionlint.yml
-        - files~=.github/.*$
+        - files~=\.github/.*\.ya?ml
```

Comment:
Per the comment immediately above, the regex should match the globs in actionlint.yml. So

```regex
\.github/(actions|workflows)/(.*\.ya?ml|actionlint\..*)

```

Reply:
Done